### PR TITLE
(fix) make [stem_group],mute a powerwindow button

### LINKED ISSUE #13749: Keyboard shortcuts for Stemcontrols are 'fishy'
### Bug Description

To help a user on the [forum ](https://mixxx.discourse.group/t/gsoc-2024-adding-stem-mixing-support/30192/26)
I tried to map the stemcontrols to my keyboard.

The [ChannelXStemY],mute does not keep the value: the value changes when pressed and when released (not only key-pressed but also key-released makes the value change)
So if you make a combination fi Ctrl+F1 to control [Channel1Stem1],mute the mute works only as long you keep the combination pressed.
The _up _down suffix does not work to the CO's [ChannelXStemY],volume.

### Version

2.6

### OS

Win

### PR DESCRIPTION
**Pull request by ronso0**
like the deck's mute button.

Fixes #13749

### CODE REVIEW

File: src/engine/channels/enginedeck.cpp [hunk 1]
```diff
@@ -84,8 +84,10 @@ EngineDeck::EngineDeck(
         // disruptive impact, so setting the default explicitly
         m_stemGain.back()->set(1.0);
         m_stemGain.back()->setDefaultValue(1.0);
-        m_stemMute.emplace_back(std::make_unique<ControlPushButton>(
-                ConfigKey(getGroupForStem(getGroup(), stemIdx), QStringLiteral("mute"))));
+        auto pMuteButton = std::make_unique<ControlPushButton>(
+                ConfigKey(getGroupForStem(getGroup(), stemIdx), QStringLiteral("mute")));
+        pMuteButton->setButtonMode(mixxx::control::ButtonMode::PowerWindow);
+        m_stemMute.emplace_back(pMuteButton);
```
Comment (Swiftb0y) [inline src/engine/channels/enginedeck.cpp:90]: almost ;)
```suggestion
        auto pMuteButton = std::make_unique<ControlPushButton>(
                ConfigKey(getGroupForStem(getGroup(), stemIdx), QStringLiteral("mute")));
        pMuteButton->setButtonMode(mixxx::control::ButtonMode::PowerWindow);
        m_stemMute.push_back(std::move(pMuteButton));
```

**Review by Swiftb0y — CHANGES_REQUESTED** [review]

File: src/engine/channels/enginedeck.cpp (hunk 1 above)
Comment (ronso0) [reply src/engine/channels/enginedeck.cpp:90]: yes :grimacing:

**Review by Swiftb0y — APPROVED** [review]
LGTM, waiting for CI. Thank you.

**Comment by JoergAtGithub** [comment]
LGTM! Thank you!

**Pull request merged**